oap-server/server-query-plugin/promql-plugin/src/main/java/org/apache/skywalking/oap/query/promql/entity/BuildInfo.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.query.promql.entity;
+
+import lombok.Data;
+
+/**
+ * Mock build info for PromQL API, sometimes the client(such as Grafana) use it to detect the version of the data-source server
+ * for different API support. SkyWalking supported basic metrics query API in Prometheus 2.45, and it is a LTS version, the API should stable.
+ */
+@Data
+public class BuildInfo {
+    private String version = "2.45.0";
+    private String revision = "";
+    private String branch = "";
+    private String buildUser = "";
+    private String buildDate = "";
+    private String goVersion = "";
```

Comment:
Could we expose this to application.yml? Maybe next time, if some configurations are required, we could fix not through code changes?